src/lib/services/identity.ts

```diff
@@ -1,8 +1,59 @@
 import { NotFound, HttpResponse } from '../error-handler'
 import env from '../../env'
+export const serviceState = {
```

Comment:
since it's a constant why not label them like

```ts
export type ServiceState = 'up' | 'down' | 'unknown'.... // ignore names
export const SERVICE_UP: ServiceState = 'up' 
```

Reply:
well, I would keep types separate from constants